example-2.0/server/model-config.json

```diff
@@ -26,6 +26,6 @@
   },
   "container": {
     "dataSource": "storage",
-    "public": true
+    "public": false
```

Comment:
I am ok with that for example 3, but for 2, it is a must. Another model could be created to state the files downloaded or uploaded but it shouldn't be pointing to the container